framework/resources/Functional/misc/describe1.e_tsv

```diff
@@ -1,4 +1,4 @@
-dfs_describe	\{
+dfs_describe.default	\{
```

Comment:
Not clear why we need this.

Reply:
This is one of the changes made by Arina:
> Default workspace name was added to the output. Example: hive -> hive.default

Reply:
Yes, we new changes when we describe schema, we show full path, including workspace.